streams/src/test/java/org/apache/kafka/streams/processor/internals/AssignedStreamTasksTest.java

```diff
@@ -38,21 +38,21 @@
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 
-public class AssignedTasksTest {
+public class AssignedStreamTasksTest {
```

Comment:
nit: change test name to `AssignedStreamsTasksTest` otherwise code coverage not picked up.